tools/ci_build/github/azure-pipelines/nuget/templates/cpu.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     AgentPool : 'Win-CPU-2019'
     ArtifactName: 'drop-nuget'
     JobName: 'Windows_CI_Dev'
-    BuildCommand:  '--build_dir $(Build.BinariesDirectory) --skip_submodule_sync --use_openmp --build_shared_lib --use_featurizers --enable_onnx_tests --use_telemetry --cmake_generator "Visual Studio 16 2019"'
+    BuildCommand:  '--build_dir $(Build.BinariesDirectory) --skip_submodule_sync --use_openmp --build_shared_lib --use_featurizers --enable_onnx_tests --use_telemetry --use_winml --cmake_generator "Visual Studio 16 2019"'
```

Comment:
I thought it will be a new pipeline and new package name for WinML?

Reply:
AFAIK, winml will always be there in the windows packages created.